tests/test_reverse_dns.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2024 Splunk, Inc.
+#
+# Use of this source code is governed by a BSD-2-clause-style
+# license that can be found in the LICENSE-BSD2 file or at
+# https://opensource.org/licenses/BSD-2-Clause
+
+import pytest
+import socket
+
+from package.etc.pylib.parser_fix_dns import FixHostnameResolver, FixFQDNResolver
+
+
+class LogMessage:
```

Comment:
Can we [import LogMessage from  syslogng](https://axoflow.com/docs/axosyslog-core/chapter-sources/python-source/python-source-logmessage/#logmessage-method-create-log-message-objects)?

Reply:
ok, understood :)